tests/integration/test_binary_op_2d.py

```diff
@@ -14,70 +14,65 @@
 #
 
 import numpy as np
+import pytest
 
 import cunumeric as num
 
+numpyX = np.array([[1, 2, 3], [4, 5, 6]])
+numpyY = np.array([[7, 8, 9], [10, 11, 12]])
 
-def test():
-    numpyX = np.array([[1, 2, 3], [4, 5, 6]])
-    numpyY = np.array([[7, 8, 9], [10, 11, 12]])
+x = pytest.fixture(lambda: num.array(numpyX))
+y = pytest.fixture(lambda: num.array(numpyY))
```

Comment:
Naming conventions are also *all* over the map in the tests. I did update a few places but stopped after I realized it would be a larger job. I would like to make a pass at standardizing some further things in the future, though.

Reply:
Also, this is an example of having to use a fixture, due to https://github.com/nv-legate/legate.core/issues/205